addons/udes_stock/models/stock_picking_type.py

```diff
@@ -12,6 +15,9 @@
 class StockPickingType(models.Model):
     _inherit = "stock.picking.type"
 
+    # Overwrite sequence_code as it is only needed for old Odoo prefixes.
+    sequence_code = fields.Char("Sequence_code", required=False)
```

Comment:
Q: Do we need to set `Sequence_code`? Should just be able to do `required=False`.

It might be worth considering adding `groups="base.no_one"` - to prevent this field from being accessible by normal users, but not really a big deal.

Reply:
This should be changed now